Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+def formElementPathVersion
+node('hi-speed') {
+    stage('plugin') {
+        dir('plugin') {
+            checkout scm
+            timeout(30) {
+                withMaven(jdk: 'Oracle JDK 1.8 (latest)',
+                        maven: 'Maven 3.2.1',
+                        mavenSettingsConfig: 'org.jenkinsci.plugins.configfiles.maven.MavenSettingsConfig1382087369423',
```

Comment:
This refers to https://jenkins.ci.cloudbees.com but it probably deserves a human-readable id.